src/lib/pages/interaction/hooks/useValidateModuleInput.ts

```diff
@@ -0,0 +1,37 @@
+import { useCallback } from "react";
+
+import {
+  useCurrentChain,
+  useExampleAddresses,
+  useValidateAddress,
+} from "lib/app-provider";
+import { isHexAddress, truncate } from "lib/utils";
+
+export const useValidateModuleInput = () => {
+  const { validateUserAddress } = useValidateAddress();
+  const {
+    chain: { bech32_prefix: prefix },
+  } = useCurrentChain();
+
+  const { user } = useExampleAddresses();
+  const truncateExampleAddr = truncate(user);
+  const errText = `Input must be address (${truncateExampleAddr} or “0x123...456) or module path (“${truncateExampleAddr}::module_name” or “0x123...456::module_name)`;
+
+  return useCallback(
+    (input: string): string | null => {
+      const inputArr = input.split("::");
```

Comment:
Let's make function for convert `address::module_name` to `[address, module_name]` in util.